src/loaders/ttf/tvgTtfLoader.cpp

```diff
@@ -148,7 +148,7 @@ static uint32_t* _codepoints(const char* text, size_t n)
             return nullptr;
         }
     }
-    utf32[i - 1] = 0;   //end of the unicdoe
+    utf32[i] = 0;   //end of the unicdoe
```

Comment:
// End of Unicode.

Reply:
Thanks, your contribution will be very helpful for ThorVG!